packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/form/ViewFormDescriptionConverter.java

```diff
@@ -234,6 +240,12 @@ private Optional<String> evaluateString(AQLInterpreter interpreter, VariableMana
         return interpreter.evaluateExpression(variableManager.getVariables(), expression).asString();
     }
 
+    private Object evaluateObject(AQLInterpreter interpreter, VariableManager variableManager, String expression) {
+        return interpreter.evaluateExpression(variableManager.getVariables(), expression)
+                .asObject()
+                .orElse(null);
+    }
```

Comment:
You don't need to create a new method for pretty much one method call used one time